frontend/src/layout/navigation/ProjectNotice.tsx

```diff
@@ -108,6 +108,15 @@ export function ProjectNotice(): JSX.Element | null {
                 children: 'Log out',
             },
         },
+        internet_connection_issue: {
+            message: 'PostHog is having trouble connecting to the internet. Please check your connection.',
```

Comment:
Maybe "connecting to the server"? There is also a low chance that we're down or an endpoint is acting up